ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelContainer/v1/panelContainer/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
```

Comment:
1. Copyright date

2. We need the panel properties as defined in AF1

https://wiki.corp.adobe.com/display/lc/Adaptive+Form+Migration#0582b34a-0fc7-4c86-8ce5-e972ca5167b6-2667176249